src/goPath.ts

```diff
@@ -11,16 +11,21 @@
 import fs = require('fs');
 import path = require('path');
 import os = require('os');
+import vscode = require('vscode');
```

Comment:
This file is used by the debug adapter via the `goDebug.ts` file. The debug adapter runs in a different process than the extension host and so will not have access to `vscode`